### PR TITLE
[IMP] oca_pre_commit_hooks: Add --list-msgs option (#44)

### LINKED ISSUE #44: Add --list-msgs similar to pylint in order to show all the current available checks
Add argument parameter `--list-msgs` and show all the checks separated for each entrypoint

### PR DESCRIPTION
**Pull request by antonag32**
Fixes #44 
Calling entry points with the argument `--list-msgs` prints out a list of all enabled messages.

### CODE REVIEW

**Comment by antonag32** [comment]
Seems to be failing due to missing unit tests (decreased coverage). Working on it as I want to find the cleanest way to test this.

**Comment by moylop260** [comment]
- The PR doesn't have issue related
- The list-msgs should be similar to update_readme method where all checks are extracted from docstring of the methods using regex
- and should show all them (without processing enabled/disabled)
- and return early similar to --help
- inspired from pylint using --list-msgs

**Comment by antonag32** [comment]
Got it, got caught up in other coding shenanigans regarding classes/reusing code last night and was derailed from the main point. 

I think the PR is good now. However should we even write unit tests for this feature?

File: src/oca_pre_commit_hooks/checks_odoo_module_po.py [hunk 1]
```diff
@@ -5,6 +5,7 @@
 from collections import defaultdict
 
 import polib
+from common import get_checks_docstring
```
Comment (moylop260) [inline src/oca_pre_commit_hooks/checks_odoo_module_po.py:8]: The `tests` folder is not part of the package itself

So, we can not import files from `tests` folder

We should move the code to package and from tests call the package method instead

Comment (moylop260) [reply src/oca_pre_commit_hooks/checks_odoo_module_po.py]: FYI I have already fixed

**Pull request merged**